jobbergate-cli/jobbergate_cli/subapps/tools.py

```diff
@@ -0,0 +1,52 @@
+"""
+Tools for the subapps.
+"""
+
+from jobbergate_cli.exceptions import Abort
+
+
+def sanitize_application_selection(
```

Comment:
I think the function name is a little confusing. It's not really sanitizing, per se.

Maybe `resolve_application_selection()` or something like that?

Reply:
Addressed by [33777a8](https://github.com/omnivector-solutions/jobbergate/pull/636/commits/33777a8ac5ad68de1637f36a5ff2d1b186458ee9).